### PR TITLE
Fix CI test.

### PR DESCRIPTION
**Pull request by dhoegh**
This is WIP towards getting the CI passing again. I have separated the commits per issue.

### CODE REVIEW

File: src/logical.jl [hunk 1]
```diff
@@ -24,7 +24,7 @@ rather ther function `Eq` is used. Here we use the unicode
 * `>=` is `\ge<tab>`
 * `>` is `\gg<tab>`
 
-"""
+=#
```
Comment (dhoegh) [inline src/logical.jl]: I have commented this out as this do not document anything and is disallow on nigthlies see: https://github.com/JuliaLang/julia/pull/13708.

**Comment by dhoegh** [comment]
@jverzani, This should be good to merge as all CI's is passing.

**Comment by jverzani** [comment]
Thanks again! Is there more to come, or can I tag a new version?

**Comment by dhoegh** [comment]
I do not have any further, for now:)

**Comment by jverzani** [comment]
Okay, thanks again!

On Sun, Nov 8, 2015 at 1:56 PM, Daniel Høegh <notifications@github.com>
wrote:

> I do not have any further, for now:)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jverzani/SymPy.jl/pull/44#issuecomment-154857499>.
>



-- 
John Verzani
Chair, Department of Mathematics
College of Staten Island, CUNY
verzani@math.csi.cuny.edu

**Comment by jverzani** [comment]
Thanks again -- it sure is nice to see the build badges say "passing."

**Pull request merged**